type.proto

```diff
@@ -127,6 +128,25 @@ message Severity {
     optional int32  priority                       = 4;
 }
 
+enum StopTimeUpdateStatus {
+    DELAYED = 0;
+    ADDED = 1;
+    DELETED = 2;
+}
+
+message StopTimeUpdate {
+    optional StopTime stop_time                     = 1;
+    optional uint64 base_departure                  = 2;
+    optional uint64 base_arrival                    = 3;
```

Comment:
I would rather have a complete `base_stop_time` in case of removal, and a complete `amended_stop_time` in case of add, but that's a little bit to much info when just delayed